rasa/validator.py

```diff
@@ -130,7 +130,12 @@ def verify_intents_in_stories(self, ignore_warnings: bool = True) -> bool:
 
     def _gather_utterance_actions(self) -> Set[Text]:
         """Return all utterances which are actions."""
-        return {
+
+        responses = {
+            response.split(RESPONSE_IDENTIFIER_DELIMITER)[0]
```

Comment:
Also, we decided to use constants with the full module path, not export them directly.